src/coreclr/tools/aot/ILCompiler/ILCompiler.csproj

```diff
@@ -5,4 +5,28 @@
   </PropertyGroup>
 
   <Import Project="ILCompiler.props" />
+
+
+  <!-- BEGIN: Workaround for https://github.com/dotnet/runtime/issues/67742 -->
+  <PropertyGroup>
+    <PublishDir>$(RuntimeBinDir)ilc-published/</PublishDir>
+    <PublishTrimmed>true</PublishTrimmed>
+    <PublishReadyToRun>true</PublishReadyToRun>
+    <PublishSingleFile>true</PublishSingleFile>
+  </PropertyGroup>
+
+  <Target Name="PublishCompiler"
```

Comment:
Is the publish going to use live-built runtime or the LKG runtime from https://github.com/dotnet/runtime/blob/main/global.json ?

If it is the later, is this going to have fallout for servicing and source build?

Reply:
It's using the LKG.

We update the LKG as part of servicing, so the expectation is that we'll ship this on top of RC1 (like we did .NET 6), and upgrade to RTM for the next servicing release (we'll be always a build behind).

Reply:
But yes, this would definitely be an issue for source build and doing https://github.com/dotnet/runtime/issues/67742 would be a ship blocker then. I have another email thread on the fate of source build. People above my paygrade need to make decisions on what to keep in 7.0.